tap_zoom/client.py

```diff
@@ -67,8 +67,11 @@ def refresh_access_token(self):
                 'grant_type': 'refresh_token'
             })
 
-        self.__access_token = data['access_token']
-        self.__refresh_token = data['refresh_token']
+        try:
+            self.__access_token = data['access_token']
+            self.__refresh_token = data['refresh_token']
+        except:
+            LOGGER.error("fFailed to refresh OAuth tokens, response from Zoom API = [{data}]")
```

Comment:
The f on your f string is outside of the `"`

Reply:
My apologies, fixed.